src/app.md

```diff
@@ -119,7 +119,22 @@ Your favorite number must be 256.
 
 [![log-badge]][log] [![env_logger-badge]][env_logger] [![cat-debugging-badge]][cat-debugging]
 
-[Write me!](https://github.com/brson/rust-cookbook/issues/61)
+```rust
+#[macro_use] extern crate log;
+extern crate env_logger;
+
+fn main() {
+    env_logger::init().unwrap();
+
+    error!("this is an error {}", "message");
```

Comment:
Would you be willing to update the example with an actual error message from `Result` obtained via std lib function call? This would make the example more realistic.